x/concentrated-liquidity/swapstrategy/one_for_zero.go

```diff
@@ -181,19 +181,6 @@ func (s oneForZeroStrategy) InitializeNextTickIterator(ctx sdk.Context, poolId u
 	return iter
 }
 
-// InitializeTickValue returns the initial tick value for computing swaps based
-// on the actual current tick.
-//
-// oneForZeroStrategy assumes moving to the right of the current square root price.
-// As a result, we use forward iterator in InitializeNextTickIterator to find the next
-// tick to the left of current. The end cursor for forward iteration is inclusive.
-// Therefore, this method is, essentially a no-op. The logic is reversed for
-// zeroForOneStrategy where we use reverse iterator and have to add one to
-// the input. Therefore, we define this method to account for different strategies.
-func (s oneForZeroStrategy) InitializeTickValue(currentTick int64) int64 {
```

Comment:
Note to reviewer: the fix for both bugs coincidentally happened to be to change `currentTick + 1` to `currentTick`, which made `InitializeTickValue` a no-op for both swap directions. Thus, removing it altogether seemed to make sense.

Reply:
Could you then also rename the third parameter here please:
https://github.com/osmosis-labs/osmosis/blob/dfcedb23a9cf340cc85df914e11d4638fa7c0141/x/concentrated-liquidity/swapstrategy/zero_for_one.go#L159